Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:latest
```

the latest tag shouldn't be used here. It's better to set it to fixed version like "17-alpine". Also an alpine image reduces the image size a lot
```suggestion
FROM node:17-alpine
```